tests/services/illumina/backup/test_backup_services.py

```diff
@@ -80,14 +78,14 @@ def test_get_archived_encryption_key_path(dsmc_q_archive_output: list[str], flow
     # THEN this method should return a path object
     assert isinstance(key_path, Path)
 
-    # THEN return the key file name
+    # THEN the latest key file name should be returned
     assert (
         key_path.name
-        == f"190329_A00689_0018_A{flow_cell_name}{FileExtensions.KEY}{FileExtensions.GPG}"
-    )
+        == f"161115_ST-E00214_0118_B{flow_cell_name}{FileExtensions.KEY}{FileExtensions.GPG}"
+    ), f'TEST FAILED: "{key_path.name}" is not the latest cryption key path'
```

Comment:
Keep it if you want it but to me this makes the test a bit harder to read.

Reply:
make this a fixture